index.js

```diff
@@ -1,2 +1,2 @@
 // Index file for npm
-module.exports = require('./Orvibo');
```

Comment:
We might need to think about this as people who are just wanting to use the framework to connect to sockets using their own software will still just want to use Orvibo.js